.gitignore

```diff
@@ -277,3 +277,7 @@ TSWLatexianTemp*
 
 # DS_Store file from Mac computers
 .DS_Store
+
+.venv
+venv
+Project_CodeNet_Python800
```

Comment:
We shouldn't gitignore this, there is already a gitignore in the src folder and we purposefully don't have this line because we use some files in there